core/eolearn/tests/test_core_tasks.py

```diff
@@ -379,47 +379,48 @@ def test_map_features(test_eopatch):
 @pytest.mark.parametrize(
     "feature,  task_input",
     [
-        ((FeatureType.DATA, "REFERENCE_SCENES"), {(FeatureType.DATA, "MOVED_BANDS"): [2, 4, 8]}),
-        ((FeatureType.DATA, "REFERENCE_SCENES"), {(FeatureType.DATA, "MOVED_BANDS"): [2]}),
-        ((FeatureType.DATA, "REFERENCE_SCENES"), {(FeatureType.DATA, "MOVED_BANDS"): (2,)}),
-        ((FeatureType.DATA, "REFERENCE_SCENES"), {(FeatureType.DATA, "MOVED_BANDS"): 2}),
+        ((FeatureType.DATA, "bands"), {(FeatureType.DATA, "MOVED_BANDS"): [2, 4, 6]}),
+        ((FeatureType.DATA, "bands"), {(FeatureType.DATA, "MOVED_BANDS"): [2]}),
+        ((FeatureType.DATA, "bands"), {(FeatureType.DATA, "MOVED_BANDS"): (2,)}),
+        ((FeatureType.DATA, "bands"), {(FeatureType.DATA, "MOVED_BANDS"): 2}),
         (
-            (FeatureType.DATA, "REFERENCE_SCENES"),
+            (FeatureType.DATA, "bands"),
             {(FeatureType.DATA, "B01"): [0], (FeatureType.DATA, "B02"): [1], (FeatureType.DATA, "B02 & B03"): [1, 2]},
         ),
-        ((FeatureType.DATA, "REFERENCE_SCENES"), {(FeatureType.DATA, "MOVED_BANDS"): []}),
+        ((FeatureType.DATA, "bands"), {(FeatureType.DATA, "MOVED_BANDS"): []}),
     ],
 )
 def test_explode_bands(
-    test_eopatch: EOPatch,
-    feature: FeatureType,
+    patch: EOPatch,
+    feature: Tuple[FeatureType, str],
     task_input: Dict[Tuple[FeatureType, str], Union[int, Iterable[int]]],
-):
+) -> None:
     move_bands = ExplodeBandsTask(feature, task_input)
```

Comment:
the 'move' naming is weird and doesn't fit this test (or the 'extract bands' test)

Reply:
Updated, name not needed

Reply:
you still use the `move` naming, e.g. `"MOVED_BANDS"`

Reply:
Renamed